src/ThemeContext/ThemeContext.js

```diff
@@ -1,2 +1,55 @@
-import {createContext} from "react";
-export const ThemeContext = createContext({});
+import React, {
+    createContext, useEffect, useState
+} from "react";
+
+import PropTypes from "prop-types";
+
+import LOCAL_STORAGE_KEYS from "../Viewer/services/LOCAL_STORAGE_KEYS";
+import {THEME_NAMES} from "./constants";
+
+
+const DEFAULT_THEME_NAME = THEME_NAMES.DARK;
+
+const ThemeContext = createContext(DEFAULT_THEME_NAME);
+
+/**
+ * Provides a theme context for its children components.
+ *
+ * @param {React.ReactNode} children
+ * @return {JSX.Element}
+ */
+const ThemeContextProvider = ({children}) => {
+    const [theme, setTheme] = useState(DEFAULT_THEME_NAME);
+
+    const changeTheme = (newTheme) => {
```

Comment:
```suggestion
    const changeTheme = (themeName) => {
```

Let's add a docstring with the parameter type.